libs/dataset_deployer.py

```diff
@@ -67,6 +67,11 @@ def upload_csv(key_name: str, csv: str, output_dir: str):
     _logger.info(f"Generated csv for {key_name}")
 
 
+def upload_json(key_name, json: str, output_dir: str): 
```

Comment:
I don't really know if there's a ton of value in using the deployer, but hey it keeps it consistent

Reply:
yeah mostly the consistency